app/screens/onboarding/slide.tsx

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {View, useWindowDimensions} from 'react-native';
+import Animated, {Extrapolate, interpolate, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+
+import {OnboardingItem} from '@typings/screens/onboarding';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    item: OnboardingItem;
+    theme: Theme;
+    scrollX: Animated.SharedValue<number>;
+    index: number;
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    image: {
+        justifyContent: 'center',
+        maxHeight: 180,
+    },
+    title: {
+        height: 100,
+        color: theme.centerChannelColor,
+        textAlign: 'center',
+        paddingHorizontal: 20,
+    },
+    fontTitle: {
+        marginTop: 32,
+        ...typography('Heading', 1000, 'SemiBold'),
+    },
+    fontFirstTitle: {
+        ...typography('Heading', 1200, 'SemiBold'),
+        paddingTop: 40,
+    },
+    widthLastSlide: {
+        paddingHorizontal: 50,
+    },
+    firstSlideInitialPosition: {
+        left: 200,
+        opacity: 0,
+    },
+    description: {
+        textAlign: 'center',
+        paddingHorizontal: 20,
+        height: 80,
+        color: changeOpacity(theme.centerChannelColor, 0.64),
+        ...typography('Body', 200, 'Regular'),
+    },
+    itemContainer: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+}));
+
+const FIRST_SLIDE = 0;
+const LAST_SLIDE = 3;
+const FIRST_LOAD_ELEMENTS_POSITION = 400;
```

Comment:
I don't quite understand what is this position, is this where do they start? Is 400 ok for tablets?

Reply:
yes, it is the position where the elements for the first slide (image, title and description) are placed so we can animate them entering sliding from right-to-left. On the other hand, not sure if 400 is ok for tablets. My initial approach was to have a phantom empty slide and manually force the slide so it animated with the same logic as the other slides, but this seemed to be more problematic. Any suggestions on how to approach this initial animation?